app/scripts/modules/core/src/search/infrastructure/infrastructure.controller.js

```diff
@@ -75,13 +72,13 @@ module.exports = angular.module('spinnaker.search.infrastructure.controller', [
     var autoNavigate = false;
 
     if (angular.isDefined($location.search().q)) {
-      this.query = $location.search().q;
+      $scope.query = $location.search().q;
```

Comment:
I broke the original search page such that the query string was no longer a query parameter in the URL.  Changes to this file unbreak it.